node/db/src/rolling/mod.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+//! The state of the Filecoin Blockchain is a persistent, directed acyclic
+//! graph. Data in this graph is never mutated nor explicitly deleted but may
+//! become unreachable over time.
+//!
+//! This module contains a concurrent, semi-space garbage collector. The garbage
+//! collector is guaranteed to be non-blocking and can be expected to run with a
+//! fixed memory overhead and require disk space proportional to the size of the
+//! reachable graph. For example, if the size of the reachable graph is 100GiB,
+//! expect this garbage collector to use 3x100GiB = 300GiB of storage.
```

Comment:
Before merging this, we should update the specs of our DO droplets.

Reply:
Agreed, 320GB is not sufficient if docker and rust toolchains need to be installed

Reply:
But we could mount a volume of ~500GB on the fly and symbol link the data folder